portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlert.testcase

```diff
@@ -74,6 +74,30 @@ definition {
 		}
 	}
 
+	@description = "Verify the text on the next line will not flow under the icon on the alert"
+	@priority = "4"
+	test CanPreventTextFromDisplayingUnderIcon {
+		task ("When the alert contains class='alert-indicator'") {
+			VerifyElementPresent(
+				key_alertType = "success",
+				locator1 = "ClayAlert#ALERT_INDICATOR");
```

Comment:
Hey @deborasoliveira could you change `VerifyElementPresent` for `AssertElementPresent`?

Reply:
It's better to have VerifyElementPresent in When steps. We don't want to have any assertions in Given or When step if possible.

Reply:
Reason is, we are more focused on the Then steps failing and we may not care if something fails in the Given or When step b/c other tests should be covering those steps if we built our SACRED model correctly.